source/release-notes.txt

```diff
@@ -10,6 +10,25 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.38.1
+----------------
+
+New Features:
+
+- Auto expand object and array field types on field add (:issue:`COMPASS-6939`).
+- Show unindexed query insight in explain plan modal (:issue:`COMPASS-6933`).
+- Show array length on array fields on documents (:issue:`COMPASS-6938`).
+- Add ctrl + tab and ctrl + shift + tab hotkeys for switching tabs.
```

Comment:
[question] Do we typically apply text styling to release notes? If so, I'd recommend using the :kbd: role per [style guide](https://www.mongodb.com/docs/meta/style-guide/style/text-formatting/#:~:text=Keyboard%20key%20combinations%2C%20names%2C%20and%20shortcuts) recommendation.

Reply:
Believe not.  At least not when it comes to notes like this listing changes with tickets.  I've gotten pushback on Internal for that before.

I wouldn't mind adopting it as a policy, but someone would need to go through the old releases and make them consistent.